### ISSUE TITLE
paper additions

### ISSUE THREAD

**Issue by nschneid** [nert-nlp/AMR-Bibliography#19]
https://www.aclweb.org/anthology/P19-1451/
http://arxiv.org/abs/1909.00352
http://arxiv.org/abs/1909.02607
http://arxiv.org/abs/1909.01568
http://arxiv.org/abs/1909.00136

**Comment by nschneid** [comment]
https://www.ijcai.org/proceedings/2019/691

**Comment by nschneid** [comment]
https://arxiv.org/abs/1909.04303

**Comment by ablodge** [comment]
```
Authors	Title	Venue	Year	Link	arXiv	    Category
Tahira Naseem, Abhishek Shah, Hui Wan, Radu Florian, Salim Roukos, Miguel Ballesteros    Rewarding Smatch: Transition-Based AMR Parsing with Reinforcement Learning    ACL    2019    https://www.aclweb.org/anthology/P19-1451        "Parsing, Transition-based, Reinforcement-Learning"
Leonardo F. R. Ribeiro, Claire Gardent, Iryna Gurevych    Enhancing AMR-to-Text Generation with Dual Graph Representations    ???        https://arxiv.org/abs/1909.00352    "Generation"

```

**Comment by ablodge** [comment]
@nschneid I've created rows here https://github.com/nert-nlp/AMR-Bibliography/compare/review but several papers are missing a venue

**Comment by nschneid** [comment]
It looks like we've been using "preprint" if the venue is not yet known.

**Comment by nschneid** [comment]
The author fields should be quoted since they have commas.

**Comment by ablodge** [comment]
Okay, but some of these from arxiv are published in EMNLP?

**Comment by ablodge** [comment]
Also, do you have any suggestions for the Category field?

**Comment by nschneid** [comment]
> Okay, but some of these from arxiv are published in EMNLP?

I've been using "preprint" to mean "only on arXiv, not yet accepted to a conference".

**Comment by nschneid** [comment]
> Also, do you have any suggestions for the Category field?

I trust your judgment!

**Comment by ablodge** [comment]
Solved by #20
